quicklogic/common/utils/flatten_layout.py

```diff
@@ -0,0 +1,515 @@
+#!/usr/bin/env python3
+"""
+This script loads VPR architecture definition and flattens all layouts defined
```

Comment:
This script probably belongs in VPR upstream, probably as a executable in [libarchfpga](https://github.com/verilog-to-routing/vtr-verilog-to-routing/tree/master/libs/libarchfpga).  You've effectively re-written the behavior of libarchfpga here, and if you have a disagreement with VPR's implementation, it could cause significant confusion.

@vaughnbetz Thoughts?

Reply:
@litghost I'm going to remove this script as it is not needed here anymore and it looks like it ended up here by mistake.

Answering your comment: Yes this is a re-implementation of the behavior of a VPR part. It is needed when you have an architecture with layout containing other constructs than `<single>` and you want to add FASM prefixes to tiles. Currently VPR provides only support for prefixing `<single>` tiles hence such a layout needs to be re-written so that it uses only those tags.

Reply:
I understand the reason such a script might exist, I'm saying re-implementing it in python, rather than just calling into the relevant VPR code seems like a bad idea.